plugins/kubernetes/src/components/KubernetesDrawer/KubernetesDrawer.tsx

```diff
@@ -56,7 +58,7 @@ const useDrawerContentStyles = makeStyles((_: Theme) =>
     options: {
       display: 'flex',
       flexDirection: 'row',
-      justifyContent: 'flex-end',
+      justifyContent: 'space-between',
```

Comment:
OK this looks fine for two things but a second button in there would end up smack dab in the middle :)

You could postpone worrying about that, no big deal. Otherwise if you're feeling adventurous you could add an empty spacer div with flex-grow between them to fill out the emptiness. Even though ... hehe that also fails on narrow screens. Ok no matter, let's worry about that another day.